sys/kern/kern_cheri_revoke.c

```diff
@@ -490,8 +543,14 @@ kern_cheri_revoke(struct thread *td, int flags,
 				PROC_UNLOCK(td->td_proc);
 
 				vm_map_lock(vmm);
+				/* Roll back some earlier state changes. */
```

Comment:
Move comment before `vm_map_lock` so it is already after a blank line?  (Or alternatively need to add a blank line?)